apple/Tests/IntegrationTests/DriversTests/EventBusDriverTests.swift

```diff
@@ -9,9 +9,10 @@ class EventBusDriverTests: DriverTest<EventBus> {
 	func test() async throws {
 		let sut = SUT()
 		
+		let expectedEvents = Array<EventKind>([.booted, .profileSaved, .factorSourceUpdated, .accountAdded, .profileSaved])
```

Comment:
Yes, needed! first we
1. update "last used on" on factor source => `[profileSaved, .factorSourceUpdated]` 
2. then we add account => `[accountAdded, .profileSaved]`

We need the correct length since we prefix in the `Task` `.prefix(expectedEvents.count) {` so expected cannot be set here => results in wrong length => we dont get all events we need. but this solution works fine :)

Reply:
oh! didn't notice the prefix, thanks for explanation!